app/views/unwithdraw/non_manage_live_history_mode.govspeak.md

```diff
@@ -0,0 +1,6 @@
+Only a GDS admin who is also a managing editor can remove the withdrawn banner and republish this content.
+
+If you think this content needs to be updated, or should not be in history mode, you need
+[to make a request to GDS in GOV.UK support](https://support.publishing.service.gov.uk/content_advice_request/new) (link opens in a new window).
+
+Read the [full guidance on creating content](https://www.gov.uk/guidance/content-design) (link opens in a new window).
```

Comment:
Since these are relatively short we could put them in the translations file which might make them feel like there is less stuff to deal with as part of this.

Reply:
This is now one template so I think is fine as is!